packages/desktop-client/src/components/reports/util.js

```diff
@@ -24,3 +24,18 @@ export function index(data, field, mapper) {
   });
   return result;
 }
+
+export function indexCF(data, field, field2, mapper) {
```

Comment:
Can you expand `CF` in the function name here? I’m not sure what it refers to, and it would be nice to help people who stumble onto the code in the future be able to easily read it.